apps/analysis_framework/schema.py

```diff
@@ -147,6 +148,10 @@ def resolve_allowed_permissions(root, info):
     def resolve_tags(root, info):
         return info.context.dl.analysis_framework.af_tags.load(root.id)
 
+    @staticmethod
+    def resolve_used_in_project_count(root, info):
+        return Project.objects.filter(analysis_framework=root.id, is_test=False).count()
```

Comment:
Let's define Dataloader.

Do we need is_test filter here? @AdityaKhatri @subinasr

Reply:
It's not necessary.

Reply:
Reiteration: Yes, we do need this filter @sudan45 @thenav56